client/app/components/ParameterValueInput.less

```
@@ -5,9 +5,12 @@
 .parameter-input {
   display: inline-block;
   position: relative;
+  width: 100%;
 
-  .@{ant-prefix}-input[type="text"] {
-    width: 195px;
+  .@{ant-prefix}-input,
+  .@{ant-prefix}-input-number,
+  .@{ant-prefix}-select {
+    width: 100%;
```

```suggestion
    min-width: 100% !important;
```
I think this fixes the all width problems 🤟

---

Thanks! :slightly_smiling_face: